### DISCUSSION TITLE
Installation failure using UTMStack ISO

### DISCUSSION THREAD

**Discussion by Pulasthi91** [utmstack/UTMStack#827] [Q&A] [answered]
I tried to install UTM stack on virtual machine with recommended specifications.
- It took more than 5 hours to install and at the end it says "utmstack backend install failed" and I get a "502 Gateway Error" i tried to access web console.  Also during the installation i did not get the option to select what to install.
and i followed the following steps as well but no luck,

"docker stack rm utmstack
Then restart the server and run the installer again."

Appreciate your help on this

**Comment by c3s4rfred** [comment]
Hi @Pulasthi91 UTMStack has a silent installation, you don't need to select anything. Please, tell us the specifications of the machine or VM used to install.

Best regards

**Comment by Pulasthi91** [comment]
16 GB RAM 
10 CPU
180 GB HD space
and I used Oracle virtual box

**Comment by c3s4rfred** [comment, marked as answer]
Hi, @Pulasthi91 check this issue, we gave some related information -> https://github.com/orgs/utmstack/discussions/807

Best regards
